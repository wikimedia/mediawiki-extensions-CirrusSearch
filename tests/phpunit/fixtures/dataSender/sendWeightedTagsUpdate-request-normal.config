{
  "config": {
      "CirrusSearchIndexBaseName": "myfixturewiki",
      "CirrusSearchUpdateShardTimeout": "1ms"
  },
  "indexSuffix": "myidxsuffix",
  "batchSize": 2,
  "docIds": [ "1", "2", "3" ],
  "tagPrefix": "my_custom_prefix",
  "tagNames": [ "my_tag_1", "my_tag_2" ],
  "tagWeights": {
    "1": {
      "my_tag_1": 750,
      "my_tag_2": 800
    },
    "2": {
      "my_tag_1": 850,
      "my_tag_2": 900
    },
    "3": {
      "my_tag_1": 950,
      "my_tag_2": 1000
    }
  }
}
