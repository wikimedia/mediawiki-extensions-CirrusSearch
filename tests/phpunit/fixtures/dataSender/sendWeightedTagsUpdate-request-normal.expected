{
    "path": "myfixturewiki_myidxsuffix\/_bulk",
    "method": "POST",
    "data": [
        [
            {
                "update": {
                    "_id": "1"
                }
            },
            {
                "script": {
                    "source": "super_detect_noop",
                    "params": {
                        "source": {
                            "weighted_tags": [
                                "my_custom_prefix\/my_tag_1|750",
                                "my_custom_prefix\/my_tag_2|800"
                            ]
                        },
                        "handlers": {
                            "weighted_tags": "multilist"
                        },
                        "_id": "1"
                    },
                    "lang": "super_detect_noop"
                }
            },
            {
                "update": {
                    "_id": "2"
                }
            },
            {
                "script": {
                    "source": "super_detect_noop",
                    "params": {
                        "source": {
                            "weighted_tags": [
                                "my_custom_prefix\/my_tag_1|850",
                                "my_custom_prefix\/my_tag_2|900"
                            ]
                        },
                        "handlers": {
                            "weighted_tags": "multilist"
                        },
                        "_id": "2"
                    },
                    "lang": "super_detect_noop"
                }
            }
        ],
        [
            {
                "update": {
                    "_id": "3"
                }
            },
            {
                "script": {
                    "source": "super_detect_noop",
                    "params": {
                        "source": {
                            "weighted_tags": [
                                "my_custom_prefix\/my_tag_1|950",
                                "my_custom_prefix\/my_tag_2|1000"
                            ]
                        },
                        "handlers": {
                            "weighted_tags": "multilist"
                        },
                        "_id": "3"
                    },
                    "lang": "super_detect_noop"
                }
            }
        ]
    ],
    "params": [],
    "contentType": "application\/x-ndjson"
}
